### PR TITLE
Release/7.0 split performance setup.

### PR DESCRIPTION
**Pull request by LoopedBard3**
Finish release/7.0 performance setup for splitting net7 from net8.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -32,20 +31,6 @@ pr:
     - scripts/benchmarks_monthly.py
     - README.md
 
-schedules:
```
Comment (cincuranet) [inline azure-pipelines.yml:35]: Is this removal intended?

**Review by cincuranet — APPROVED** [review]

**Comment by LoopedBard3** [comment]
This doesn't change the tests themselves, just sets the pr and ci triggers to include release/7.0 on this branch.
